pynamodb/attributes.py

```diff
@@ -118,11 +119,13 @@ def _serialize(self, value):
         return {ATTR_TYPE_MAP[self.attribute.attr_type]: self.attribute.serialize(value)}
 
 
-class AttributeContainer(object):
+class AttributeContainerMeta(type):
 
-    _dynamo_to_python_attrs = None
+    def __init__(cls, name, bases, attrs):
+        super(AttributeContainerMeta, cls).__init__(name, bases, attrs)
+        AttributeContainerMeta._initialize_attributes(cls)
 
-    @classmethod
+    @staticmethod
```

Comment:
This function is now in the metaclass, so the `cls` here is the class being created, just like in `__init__`.